servers/rendering/renderer_rd/shaders/forward_mobile/scene_forward_mobile.glsl

```diff
@@ -1830,4 +1992,12 @@ void main() {
 #endif //MODE_MULTIPLE_RENDER_TARGETS
 
 #endif //MODE_RENDER_DEPTH
+
+#ifdef MODE_RENDER_MOTION_VECTORS
+	vec3 ndc = screen_position.xyz / screen_position.w;
+	ndc.y = -ndc.y;
```

Comment:
This shouldn't be necessary, have you validated the output against the output from the Forward+ Renderer? My guess is that this backwards